pkg/features/kube_features.go

```diff
@@ -204,6 +210,7 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ManagedJobsNamespaceSelector:        {Default: true, PreRelease: featuregate.Beta},
 	LocalQueueMetrics:                   {Default: false, PreRelease: featuregate.Alpha},
 	LocalQueueDefaulting:                {Default: false, PreRelease: featuregate.Alpha},
+	BestFitTAS:                          {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
I would actually suggest to make it directly to Beta in 0.11, as it seems much more sensible than the current behavior for most users. Then we would graduate to GA and drop the gate in 0.13 probably.

Reply:
Discussed offline, and we decided to keep it alpha but set to deprecated already and drop in v0.13